packages/ripple-nuxt-tide/modules/webform/components/TideContentRating.vue

```diff
@@ -250,7 +250,11 @@ $app-content-text-color: rpl-color('dark_neutral') !default;
   }
 
   .app-button--cancel {
-    background-color: rpl-color('danger');
+    background-color: $rpl-button-danger-background-color;
```

Comment:
This is nice to see you fix it. 👍  All color in components need a meaningful var instead of directly use color.